src/Map/composer.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "symfony/ux-map",
+    "type": "symfony-bundle",
+    "description": "Easily embed interactive maps in your Symfony application",
+    "keywords": [
+        "symfony-ux"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Hugo Alliaume",
+            "email": "hugo@alliau.me"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Symfony\\UX\\Map\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Symfony\\UX\\Map\\Tests\\": "tests/"
+        }
+    },
+    "require": {
+        "php": ">=8.1",
+        "symfony/stimulus-bundle": "^2.18.12"
+    },
+    "require-dev": {
+        "symfony/asset-mapper": "^6.4|^7.0",
+        "symfony/framework-bundle": "^5.4|^6.0|^7.0",
+        "symfony/phpunit-bridge": "^5.2|^6.0|^7.0",
+        "symfony/twig-bundle": "^5.4|^6.0|^7.0",
+        "symfony/var-dumper": "^5.4|^6.0|^7.0"
```

Comment:
"^6.4|^7.0"

Reply:
I went for that aswell in the 2nd iteration!